WTWSMS/decisions/WtWSMS_realm_decisions.txt

```diff
@@ -413,11 +431,12 @@ decisions = {
 			OR = {
 				religion_group = christian
 				religion_group = pagan_group
+				has_romance_culture_trigger = yes
+				culture = roman
 			}
 			age = 16
 			prisoner = no
 			NOT = { trait = incapable }
-			has_romance_culture_trigger = yes
```

Comment:
The Charlemagne empire will be added with specific flavour, while you wanted to re-enable vanilla content which wasn't adapted.